terraform/cross-region-infra/dynamodb-global-tables.tf

```diff
@@ -0,0 +1,168 @@
+######################################################
+# Video global tables and replicas
+######################################################
+resource "aws_dynamodb_table" "videos_primary" {
+  # TODO: Got an error " A provider configuration reference must not be given in quotes" error when trying to reference like this 
+  # provider = "aws.${local.primary_region}"
+  provider = aws.eu-central-1
```

Comment:
(Question) Do we have better option?

This [answer](https://discuss.hashicorp.com/t/defining-provider-aliases-with-string-interpolation-not-working-in-terraform-0-12/1569/4) says that it is not possible to use dynamic value.